tests/test_core.py

```diff
@@ -229,7 +250,7 @@ def __test(ref_power, clip):
             if six.callable(ref_power) or ref_power >= 0.0:
                 tf = __test
             else:
-                tf = raises(librosa.ParameterError)(__test)
+                tf = raises(ValueError)(__test)
```

Comment:
I think this test is failing on travis because you reverted the exception type here (and also line 558 below).